plans/e2e.fmf

```diff
@@ -0,0 +1,59 @@
+discover:
+  how: fmf
+  test: e2e
+adjust+:
+  - when: arch == x86_64 or arch == aarch64
+    provision:
+      hardware:
+        cpu:
+          processors: ">= 2"
+        memory: ">= 6 GB"
+        virtualization:
+          is-supported: true
+prepare:
+  - how: shell
+    script: |
+      source /etc/os-release
+      if [[ "$ID" == "centos" ]]; then
+          # EPEL for genisoimage
+          dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+      fi
+  - how: install
+    package:
+      - ansible-core
+      - firewalld
+      - podman
+      - skopeo
+      - jq
+      - openssl
+      - qemu-img
+      - qemu-kvm
+      - libvirt
+      - virt-install
+      - genisoimage
+  - how: shell
+    script: ansible-galaxy collection install https://ansible-collection.s3.amazonaws.com/ansible-posix-1.5.4.tar.gz https://ansible-collection.s3.amazonaws.com/community-general-8.5.0.tar.gz
+execute:
+  how: tmt
+
+/to-existing-root:
+  summary: Run bootc install to-existing-root and bootc switch test locally (nested)
+  environment+:
+    TEST_CASE: to-existing-root
+  discover+:
+    test:
+      - /to-existing-root
+  adjust+:
+    - when: arch == ppc64le
+      enabled: false
```

Comment:
This syntax is YAML sorcery to me...is there any description of what the `/` and `+` do in particular?

Reply:
When configure tmt plan or test, it supports regular expression, starting with `/` to avoid regular expression, but just a string match. The `/` comes from plan or test name, for example:
```
➜ tmt test ls
/tests/e2e/e2e/to-disk
/tests/e2e/e2e/to-existing-root
/tests-integration/build-image
```

The `+` means adding addition configuration based on `top level` configuration. In this case, The top level has `adjust` configured already.